bindings/nodejs/CHANGELOG.md

```diff
@@ -19,6 +19,12 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ### Security -->
 
+## 1.1.6 - 2025-04-24
```

Comment:
I don't remember but isn't there an actual file where we need to bump this for nodejs?